torch/_prims/nvfuser_prims.py

```diff
@@ -0,0 +1,301 @@
+# Module for defining "primitive" operations executable by the nvFuser. This
+# list exists to decouple main set of primitives from the ones that provide a
+# lowering of the op to nvFuser’s Python interface. Mostly torch.ops.nvprims is
+# a subset of the primitives in torch.ops.prims, but some additional primitives
+# can be added in the future for the corresponding higher-level torch/aten
+# functions.
```

Comment:
do you want to list those nvfuser-specific prims?

Reply:
In the near future, it's var_mean and probably batch_norm, instance_norm.